<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iterate CRISPR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Key: </string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="keyLabel">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Experiment: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="experLabel">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Export Database</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportButton">
             <property name="toolTip">
              <string>Export entire traces database (CSV format)</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/document-export-table.png</normaloff>:/document-export-table.png</iconset>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="hintLabel">
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: rgb(255, 255, 0);
	border-radius: 3px;
	padding: 2px;
}</string>
          </property>
          <property name="text">
           <string>Hint:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hintText">
          <property name="text">
           <string>Double click primer to select in assembler</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="selectAllButton">
          <property name="toolTip">
           <string>Select all primers</string>
          </property>
          <property name="text">
           <string>Select All</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/edit-select-all.png</normaloff>:/edit-select-all.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectNoneButton">
          <property name="toolTip">
           <string>Clear Selection</string>
          </property>
          <property name="text">
           <string>Select None</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/edit-clear-list.png</normaloff>:/edit-clear-list.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"> QListWidget::item {
     padding: 5px;
 }

QListWidget {
     show-decoration-selected: 0; 
 }

 QListWidget::item:selected {
   /* background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);*/
   color: #000;
    background-color: #bfd9ff;
 }

</string>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="recalculateButton">
             <property name="toolTip">
              <string>Redo the assembly</string>
             </property>
             <property name="text">
              <string>Recalculate</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/task-recurring.png</normaloff>:/task-recurring.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="finishedButton">
             <property name="text">
              <string>Save as Finished</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/task-complete.png</normaloff>:/task-complete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="unfinishedButton">
             <property name="text">
              <string>Save as Unfinished</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/task-ongoing.png</normaloff>:/task-ongoing.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="skipButton">
             <property name="toolTip">
              <string>Skip this entry</string>
             </property>
             <property name="text">
              <string>Skip</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/arrow-right-double.png</normaloff>:/arrow-right-double.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="toolTip">
              <string>Stop the program</string>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>selectAllButton</tabstop>
  <tabstop>selectNoneButton</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>recalculateButton</tabstop>
  <tabstop>finishedButton</tabstop>
  <tabstop>unfinishedButton</tabstop>
  <tabstop>skipButton</tabstop>
  <tabstop>stopButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
